jdaviz/configs/imviz/tests/test_catalogs.py

```diff
@@ -240,27 +254,34 @@ def test_offline_ecsv_catalog(imviz_helper, image_2d_wcs, tmp_path):
     assert catalogs_plugin.number_of_results == n_entries
     assert len(imviz_helper.app.data_collection) == 2  # image + markers
 
-    catalogs_plugin.clear()
+    catalogs_plugin.clear_table()
 
     assert not catalogs_plugin.results_available
     assert len(imviz_helper.app.data_collection) == 2  # markers still there, just hidden
 
-    catalogs_plugin.clear(hide_only=False)
+    catalogs_plugin.clear_table(hide_only=False)
     assert not catalogs_plugin.results_available
     assert len(imviz_helper.app.data_collection) == 1  # markers gone for good
 
     assert imviz_helper.viewers['imviz-0']._obj.state.x_min == -0.5
     assert imviz_helper.viewers['imviz-0']._obj.state.x_max == 9.5
     assert imviz_helper.viewers['imviz-0']._obj.state.y_min == -0.5
     assert imviz_helper.viewers['imviz-0']._obj.state.y_max == 9.5
+    # Re-populate the table with a new search
+    out_tbl = catalogs_plugin.search()
+    assert len(out_tbl) > 0
+    # Ensure at least one row is selected before zooming
+    catalogs_plugin.table.selected_rows = [catalogs_plugin.table.items[0]]
+    assert len(catalogs_plugin.table.selected_rows) > 0
 
     # test the zooming using the default 'padding' of 2% of the viewer size
     # around selected points
     catalogs_plugin.zoom_to_selected()
-    assert imviz_helper.viewers['imviz-0']._obj.state.x_min == -0.19966
-    assert imviz_helper.viewers['imviz-0']._obj.state.x_max == 0.20034000000000002
-    assert imviz_helper.viewers['imviz-0']._obj.state.y_min == 0.8000100000000001
-    assert imviz_helper.viewers['imviz-0']._obj.state.y_max == 1.20001
+    assert_allclose(imviz_helper.viewers['imviz-0']._obj.state.x_min, -0.19966, rtol=1e-1)
+    assert_allclose(imviz_helper.viewers['imviz-0']._obj.state.x_max,
+                    0.20034000000000002, rtol=1e-1)
```

Comment:
Why did this go from exact comparison to really loose allclose with 10% tolerance?

```suggestion
    assert_allclose(imviz_helper.viewers['imviz-0']._obj.state.x_max, 0.20034000000000002, rtol=1e-1)
```